firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/HAL/STM32/DigitalOutput.h

```diff
@@ -6,24 +6,26 @@
 
 #pragma once
 
-#include "Pufferfish/HAL/HAL.h"
+#include "Pufferfish/HAL/Interfaces/DigitalOutput.h"
+#include "Pufferfish/HAL/STM32/HAL.h"
```

Comment:
Let's do remove the inclusion of `HAL.h`, as `HAL.h` includes this file already. It looks like the original code made a mistake here, so we can use this PR to fix the issue.